### PR TITLE
Added encrypted credential files and enable integration tests

### PR DESCRIPTION
**Pull request by renkelvin**

### CODE REVIEW

**Comment by paulb777** [comment]
Looks like an issue in https://travis-ci.org/firebase/firebase-ios-sdk/jobs/496638822

**Comment by paulb777** [comment]
Perhaps something from https://stackoverflow.com/questions/34304570/how-to-resolve-the-evp-decryptfinal-ex-bad-decrypt-during-file-decryption?

**Comment by renkelvin** [comment]
I think the issue is that you can't encrypt multiple files. You have to compress then encrypt. https://github.com/travis-ci/travis-ci/issues/8759

**Comment by paulb777** [comment]
Looks like encryption is working now, but I suspect something in the `HEADER_SEARCH_PATHS` in the Sample project is blocking the CocoaPods overrides to find the FirebaseCore headers.

**Comment by renkelvin** [comment]
Encryption should be working now.

File: .travis.yml [hunk 1]
```diff
@@ -290,3 +290,4 @@ jobs:
 branches:
   only:
     - master
+    - enable-integration-tests
```
Comment (paulb777) [inline .travis.yml]: Revert this.

**Pull request merged**